Source/WebKit/UIProcess/AuxiliaryProcessProxy.h

```diff
@@ -277,6 +277,15 @@ class AuxiliaryProcessProxy
     static RefPtr<WebCore::SharedBuffer> fetchAudioComponentServerRegistrations();
 #endif
 
+    virtual void didFinishProcessInitialization()
```

Comment:
Why is this virtual? I don't see an override

Reply:
Good point! I'll make it non-virtual in the next patch.